numpy/lib/arraysetops.py

```diff
@@ -770,6 +770,7 @@ def setdiff1d(ar1, ar2, assume_unique=False):
     """
     if assume_unique:
         ar1 = np.asarray(ar1).ravel()
+        ar1.sort()
```

Comment:
I was worried this might mutate the input array, may be safer with `np.sort`. If so, should test for that too since all green already.